collection-scripts/gather

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Resource List
+resources=()
+
+# Operator Resources
+resources+=(clusteroperators)
+
+# Certificate Resources
+resources+=(csr)
```

Comment:
@deads2k I assume given the threads you have included me in this is a thing we want to collect.

Reply:
I updated this to be plural thus why its out dated.

Reply:
```suggestion
resources+=(certificatesigningrequests)
```
`csr` is just a short name, so the plural form is `certificatesigningrequests`:
```console
[sanchezl@monolith must-gather]$  ./openshift-must-gather inspect csrs --base-dir CSRS
Error: the server doesn't have a resource type "csrs"
[sanchezl@monolith must-gather]$  ./openshift-must-gather inspect certificatesigningrequests --base-dir CSR
2019/04/01 21:15:22 Finished successfully with no errors.
[sanchezl@monolith must-gather]$ tree CSR/
CSR/
└── cluster-scoped-resources
    └── certificates.k8s.io
        └── certificatesigningrequests
            ├── csr-29q65.yaml
            ├── csr-49n9f.yaml
            └── csr-9qwh4.yaml

3 directories, 3 files
[sanchezl@monolith must-gather]$ 
```

Reply:
Fixed